src/web/layouts/CommentLayout.tsx

```diff
@@ -465,7 +465,12 @@ export const CommentLayout = ({
                     {showComments && (
                         <Section sectionId="comments">
                             <Flex>
-                                <div id="comments-root" />
+                                <div
+                                    id="comments-root"
+                                    className={css`
+                                        width: 100%;
+                                    `}
+                                />
```

Comment:
I'm not sure we want to start applying styles to these root marker divs - I worry that could confuse thing